doc/usecases/elektra_web/UC_finding_keys.md

```diff
@@ -0,0 +1,20 @@
+# Use Case: Finding keys
+
+## Summary
+
+Title: Finding keys
+Scope: Configuration
+Level: User Goal
+Actors: User (usually a sysadmin)
+Brief: User searches for a key by filtering the key database.
+
+## Scenarios
+
+Precondition: View configuration of an instance
+Main success scenario: User searches for a key by entering information (its path
+  or data) in a field. The whole key database is filtered according to the
```

Comment:
data -> value.

Search for key is much faster, what about first show by key and continue searching for values in background?